MatteoCellucci adds hometown of Rome, Italy

matteocellucci opened the pull request:
- Is anyone interested in best historical monuments?

matteocellucci committed:
Add restaurants to Rome file

cstumpfel commented on matteocellucci-rome.md:
```diff
@@ -0,0 +1,9 @@
+# Rome, Italy
+
```
Is this place in Kansas?

matteocellucci committed:
Improves geolocation

Remove boring stuff like Nautre beauty or Attractions. No body cares of them :devil:
